core/src/main/java/org/apache/carbondata/core/metadata/schema/table/column/CarbonDimension.java

```diff
@@ -51,6 +51,16 @@
    */
   private int complexTypeOrdinal;
 
+  /**
+   * to store the Ordinal of the Complex Parent Column.
+   */
+  private int parentOrdinal = -1;
```

Comment:
Please remove it as we can get the information from `complexParentDimension`

Reply:
Removed